.github/workflows/check-clog-linux.yml

```diff
@@ -0,0 +1,31 @@
+name: CLOG
+
+on:
+  push:
+    branches:
+    - main
+    - release/*
+  pull_request:
+    branches:
+    - main
+    - release/*
+
+jobs:
+  validate:
+    name: Validate(Linux)
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - run: |
+         sudo apt-add-repository ppa:lttng/stable-2.12
+         sudo apt-get update
+         sudo apt-get install -y liblttng-ust-dev lttng-tools build-essential
+
+    - name: Build CLOG and run Tests
```

Comment:
Maybe a good idea to separate build and test so it's easier to know what failed when looking at the failed job name?

Reply:
I did this intentionally in hopes having them lumped makes "testing at home" easier; I'm not against flipping back down the road, but could we try this for a while?  I'm wanting to lean a lot more on examples that serve as CI/CD tests and would like to make deving smoother than CI/CD

Reply:
Sure. We can look into it more later.